src/Components/Web.Extensions/src/Microsoft.AspNetCore.Components.Web.Extensions.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

A little nit-picky, if we aren't using static web assets, this can be 

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```

---

I left this as-is because I figured components like file upload, modal, etc. are also eventually going in this package, but I guess it doesn't hurt to change it to `<Project Sdk="Microsoft.NET.Sdk">` and modify it again later if needed.